test/integ/quote-classic.test.ts

```diff
@@ -807,6 +827,62 @@ describe('quote', function () {
             }
           });
 
+          it.each(greenlistTokensPairs)(`$tokenIn.symbol -> $tokenOut.symbol`, async (tokenIn, tokenOut) => {
+            it.each([true, false])(`sendPortionEnabled = $sendPortionEnabled`, async (_) => {
+              // Arrange:
+              // - token amount from swapper
+              // - greenlist token pairs
+              // - parametrize on sendPortionEnabled
+              getAmount(1, type, tokenIn.symbol!, tokenOut.symbol!, '100');
+
+              // Act:
+              // - call classic quote
+
+              // Assert:
+              // - check if the response is 200
+              // - check if the response contains the methodParameters
+              // - check if the response contains portion-related payloads if sendPortionEnabled = true
```

Comment:
how do you know here whether or not `sendPortionEnabled = true` if you're not passing in a param into this function? We can do something like we do for `tradeType` where we iterate over the values:
```
for(const tradeType of [exactIn, exactOut]) {
   it( '' , async (tradeType) => { } )
}
```

Reply:
don't need to address since this is only skeleton for a test but can refactor the `it.each()` format above to pass in the boolean value to each nested test